quiz.rb

```diff
@@ -48,7 +69,20 @@ def ac_needed current_temp, ac_working, desired_temp
 # and "fizz buzz" if the number is evenly divisible by both 3 and 5
 # note the space in "fizz buzz"
 
-# your code starts here
+def fizzbuzz num
+
+  if num % 5 === 0 && num % 3 === 0
```

Comment:
The ruby `===` operator is different than the javascript `===` operator. when working with ruby, use `==` instead